### DISCUSSION TITLE
list of missing icons

### DISCUSSION THREAD

**Discussion by brigazvi** [akiraux/Akira#626] [General]
in #358 there is 'create missing symbolic icons' task. couldn't find what missing so here I am creating this issue.
@Alecaddd?

**Comment by Suzie97** [comment]
Yes, I would be glad to make those icons and contribute. It'll be handy to have a list.

**Comment by Alecaddd** [comment]
I need to go through the current UI and see what's missing and what's needed.
I'll convert this issue into a discussion as it suits it better.
